test/FunctionalTestUtils/project.json

```diff
@@ -14,16 +13,23 @@
   },
   "testRunner": "xunit",
   "frameworks": {
-    "netstandardapp1.5": {
+    "net451": {
+      "compilationOptions": { "define": [ "net451" ] },
```

Comment:
Shouldn't need this. `dotnet` should add a `NET451` preprocessor variable for you.